src/main/java/seedu/address/timetable/TimetableUrlParser.java

```diff
@@ -1,6 +1,7 @@
 package seedu.address.timetable;
 
 public class TimetableUrlParser {
+    private static final String nusModsProtocolAndHost = "https://nusmods.com/timetable/";
```

Comment:
According to the CS2103T coding conventions, all constants must be in UpperCase and use underscore to separate words.

![image](https://user-images.githubusercontent.com/58943071/98289605-61931b00-1fe3-11eb-9165-bbf51a2032e2.png)

Since `nusModsProtocolAndHost` is a final variable, it is a constant and should hence be named `NUSMODS_PROTOCOL_AND_HOST`.

Reply:
Amended!